x/interchainstaking/keeper/grpc_query.go

```diff
@@ -116,7 +116,7 @@ func (k *Keeper) DepositAccount(c context.Context, req *types.QueryDepositAccoun
 
 	zone, found := k.GetZone(ctx, req.GetChainId())
 	if !found {
-		return nil, status.Error(codes.NotFound, fmt.Sprintf("no zone found matching %s", req.GetChainId()))
```

Comment:
Are the status errors, returning codes linked to the HTTP response code? If so, do these need to remain wrapped?

Reply:
this might be true, good catch. I reverted changes of status'ed errors, should be good to merge now